b/better-ajv-errors/better-ajv-errors_ubi8.4.sh

```diff
@@ -0,0 +1,49 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : better-ajv-errors
+# Version       : v0.5.7
+# Source repo   : https://github.com/atlassian/better-ajv-errors.git
+# Tested on     : UBI 8.4
+# Script License: Apache 2.0 License
+# Maintainer    : Mahima Gaikwad <mahima.gaikwad@ibm.com>
+#
```

Comment:
Please add the additional fields that have been introduced recently - Language, Travis-Check etc

Reply:
Added Language, Travis-Check fields